model/src/main/java/com/ge/predix/acs/rest/AttributeConnector.java

```diff
@@ -0,0 +1,37 @@
+package com.ge.predix.acs.rest;
+
+import java.util.Set;
+
+public class AttributeConnector {
+    
+    private boolean isActive;
+    
+    private int maxCachedIntervalMinutes;
+    
+    private Set<AttributeAdapterConnection> adapters;
+
+    public boolean getIsActive() {
```

Comment:
To unblock other pull requests that depend on this story I would handle persisting isActive as a separate story.